core/datastore/src/main/java/com/google/samples/apps/nowinandroid/core/datastore/NiaPreferencesDataSource.kt

```diff
@@ -39,20 +39,22 @@ class NiaPreferencesDataSource @Inject constructor(
                     null,
                     ThemeBrandProto.THEME_BRAND_UNSPECIFIED,
                     ThemeBrandProto.UNRECOGNIZED,
-                    ThemeBrandProto.THEME_BRAND_DEFAULT -> ThemeBrand.DEFAULT
+                    ThemeBrandProto.THEME_BRAND_DEFAULT,
+                    -> ThemeBrand.DEFAULT
```

Comment:
Don't love the new wrapping for `when` introduced here, but don't know if there's a solution yet